x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.tsx

```diff
@@ -100,6 +104,40 @@ export const RelevanceTuningForm: React.FC = () => {
             </EuiAccordion>
           </EuiPanel>
         ))}
+        <EuiSpacer />
+        {filteredSchemaFieldsWithConflicts.length > 0 && (
+          <>
+            <EuiTitle size="s" data-test-subj="DisabledFieldsSection">
+              <h3>
+                {i18n.translate(
+                  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.disabledFields.title',
+                  {
+                    defaultMessage: 'Disabled fields',
+                  }
+                )}
+              </h3>
+            </EuiTitle>
+            <EuiSpacer />
+            <EuiFlexGroup direction="column">
+              {filteredSchemaFieldsWithConflicts.map((fieldName) => (
+                <EuiFlexItem key={fieldName} style={{ alignItems: 'flex-start' }}>
+                  <EuiTitle size="xs">
+                    <h4 data-test-subj="DisabledField">{fieldName}</h4>
+                  </EuiTitle>
+                  <EuiHealth color="warning">
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.disabledFieldsExplanationMessage',
+                      {
+                        defaultMessage: 'Inactive due to field-type conflict',
+                      }
+                    )}
+                  </EuiHealth>
+                </EuiFlexItem>
+              ))}
+            </EuiFlexGroup>
+            <EuiSpacer />
+          </>
```

Comment:
[optional] I know this is ported as-is from the standalone UI, but if no objections I have a small set of UI changes that I think makes this look a bit nicer/more coherent with the rest of the page:

### Before
<img width="569" alt="" src="https://user-images.githubusercontent.com/549407/109216803-de502d80-7769-11eb-911c-fe709749e37b.png">

### After
<img width="570" alt="" src="https://user-images.githubusercontent.com/549407/109216698-bb257e00-7769-11eb-8e08-81a262542a29.png">

```suggestion
            <EuiSpacer size="s" />
            {filteredSchemaFieldsWithConflicts.map((fieldName) => (
              <EuiPanel key={fieldName} className="relevanceTuningForm__panel">
                <EuiTitle size="xs">
                  <h4 data-test-subj="DisabledField">{fieldName}</h4>
                </EuiTitle>
                <EuiHealth color="warning">
                  {i18n.translate(
                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.disabledFieldsExplanationMessage',
                    {
                      defaultMessage: 'Inactive due to field-type conflict',
                    }
                  )}
                </EuiHealth>
              </EuiPanel>
            ))}
          </>
```